src/Panes/AppearancePane.vala

```diff
@@ -31,47 +31,88 @@ public class PantheonTweaks.Panes.AppearancePane : Categories.Pane {
     private int controls_index = 0;
 
     public AppearancePane () {
-        base (_("Appearance"), "applications-graphics");
+        base (
+            _("Appearance"), "applications-graphics",
+            _("Change the theme and button layout of windows. Changing theme may cause visibility issue.")
+        );
     }
 
     construct {
         var theme_label = new Granite.HeaderLabel (_("Theme Settings"));
-        var theme_box = new Widgets.SettingsBox ();
 
+        var gtk_label = new SummaryLabel (_("GTK+:"));
         var gtk_store = Util.get_themes_store ("themes", "gtk-3.0", InterfaceSettings.get_default ().gtk_theme, out gtk_index);
-        gtk_combobox = theme_box.add_combo_box (_("GTK+"));
+        gtk_combobox = new ComboBox ();
         gtk_combobox.set_model (gtk_store);
 
+        /// TRANSLATORS: The two "%s" represent the paths to put custom icons
+        var gtk_info = new DimLabel (_("To show custom themes here, put them in %s.").printf (
+            "~/.local/share/themes/<theme-name>/gtk-3.0"
+        ));
+
+        var icon_label = new SummaryLabel (_("Icons:"));
         var icon_store = Util.get_themes_store ("icons", "index.theme", InterfaceSettings.get_default ().icon_theme, out icon_index);
-        icon_combobox = theme_box.add_combo_box (_("Icons"));
+        icon_combobox = new ComboBox ();
         icon_combobox.set_model (icon_store);
 
+        /// TRANSLATORS: The two "%s" represent the paths to put custom icons
```

Comment:
nit? :) The comment would be a like, just like the cake is
```suggestion
        /// TRANSLATORS: The "%s" represent the path where custom icons are installed
```

Reply:
My English is bad :sweat_smile:

Should the verb "represent" be now "**represents**" because the subject is the third-person singular?

Reply:
Yes you are correct, it should be represents :) And no worries, English is also not my first language, but this is what PRs and discussions are for :)